articles/azure/azs-how-configure-mars-backups-powershell.md

```diff
@@ -0,0 +1,91 @@
+description: Provides details on how to install and configure the Microsoft Azure Recovery Services (MARS) agent to backup files and folders on Azure Stack
+services: azure-stack
+author: Daniel Brennand
+reviewer:
+lastreviewed:
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Back up files and folders - PowerShell
+toc_fullpath: Users/How To/azs-how-configure-mars-backups-powershell.md
+toc_mdlink: azs-how-configure-mars-backups-powershell.md
+---
+
+# How to back up files and folders on Azure Stack VMs using PowerShell
+
+# Overview
+
+This article explains how to setup the Microsoft Azure Recovery Services (MARS) agent to backup files and folders from Azure Stack VMs to Recovery Services vaults.
+
+Recovery Services vaults store all backups and recovery points you create over time, and contains the backup policy applied to backed up machines.
+
+## Official documentation
+
+[MARS agent support matrix](https://docs.microsoft.com/en-us/azure/backup/backup-support-matrix-mars-agent)
+
+[Back up windows machines using MARS](https://docs.microsoft.com/en-us/azure/backup/backup-configure-vault)
+
+[MARS agent backup process](https://docs.microsoft.com/en-us/azure/backup/backup-architecture#architecture-direct-backup-of-on-premises-windows-server-machines-or-azure-vm-files-or-folders)
+
+## Prerequisites
+
+To complete the steps in this article, you must have appropriate access to a subscription in both public Azure and Azure Stack portals.
+
+> [!IMPORTANT]
+> The MARS agent is **only** supported on Windows VMs. Linux is **not** supported.
+
+## Setup the MARS agent using PowerShell
+
+The script used in this article can be found [here](https://github.com/UKCloud/AzureStack/blob/master/Users/Extensions/Windows/AzureBackupConfig.ps1). It provides docstrings on additional parameters which are not used in this article.
+
+### Declare variables
+
+Enter details below to provide values for the variables in the scripts in this article:
+
+| Variable name   | Variable description                                               | Input            |
+|-----------------|--------------------------------------------------------------------|------------------|
+| \$ClientID    | The application ID of a service principal with contributor permissions on Azure.                 | <form oninput="result.value=clientid.value" id="clientid" style="display: inline;"><input type="text" id="clientid" name="clientid" style="display: inline;" placeholder="00000000-0000-0000-0000-000000000000"/></form> |
+| \$ClientSecret    | The password of the service principal specified in the ClientId parameter.                 | <form oninput="result.value=clientsecret.value" id="clientsecret" style="display: inline;"><input type="text" id="clientsecret" name="clientsecret" style="display: inline;" placeholder="ftE2u]iVLs_J4+i-:q^Ltf4!&{!w3-%=3%4+}F2jk|]="/></form> |
```

Comment:
I think the pipe symbol (|) in the placeholder may be messing with the input box?

Reply:
Removing the pipe.